src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public String getSalaryInfo(String[] names, String[] data,
+                                       String dateFrom, String dateTo) {
+        String[] countData = new String[data.length];
+        int[] wages = new int[names.length];
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

See [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field)